.gitlab-ci.yml

```diff
@@ -25,6 +25,8 @@ stages:
   - test
   - build
   - post-build-test
+  - build-pr
+  - chaos
```

Comment:
`kubernetes` stage suits for it

Reply:
bump

Reply:
```kubernetes``` stage is taken now:)
Are you suggesting to merge the ```build-pr``` into ```kubernetes```?

Reply:
Gitlab-ci runs in order of stages right?
I think the stage should be before ```publish```.

Reply:
please move your jobs from `docker` stage to `build` and `chaos` -> `post-build-test`

Reply:
docker needs to run after ```build-linux-substrate```
And ```chaos``` stage is !== ```post-build-test```
I think this could be left as it is

Reply:
OK then, `publish` stage for the image build and `kubernetes` for chaos suit and fit better. By introducing the new stages you slower the whole pipeline: all stages are executing one after another.

Reply:
I think its fine
1. Chaos related stages only triggered by certain condition
2. If enabled, should have some chaos testing stages before ```publish```